### PR TITLE
[UK] Add `BBCode`

### PR DESCRIPTION
**Pull request by Aniterin**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/BBCode/uk.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
+# BBCode
+
+**BBCode** це [Мова розмітки](https://uk.wikipedia.org/wiki/Мова_розмітки_даних) яка використовується для спілкування на форумах і, по великому рахунку, більшість форумів в інтернеті. Використовується для урізноманітнення тексту, вона зроблена з тегів для форматування, додавання атрибутів, вкладень, і т.д. Вона використовується повсюди на сайті osu!: в постах на форумах, іменах і описах профілів, і описах мап.
+
+![Редактор публікацій](img/editor.jpg?1 "Редактор публікацій")
+
+## Швидке введення
+
+При натисканні на кнопку форматування без виділення тексту, буде створена пара тегів навколо курсора в редакторі. При натисканні на кнопку з виділеним текстом теги будуть створені навколо вибраного тексту.
```
Comment (endeshnik) [inline wiki/BBCode/uk.md:9]:
```suggestion
При натисканні на кнопку форматування без виділення тексту буде створена пара тегів навколо курсора в редакторі. При натисканні на кнопку з виділеним текстом теги будуть створені навколо вибраного тексту.
```

File: wiki/BBCode/uk.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+
+Користувачі, які хочуть використовувати разом різні теги можуть накладати їх один поверх іншого, Але, порядок накладання **повинен бути в рамках дозволенного**. При неправильному використанні це може відмінити форматування.
```
Comment (endeshnik) [inline wiki/BBCode/uk.md:11]:
```suggestion
Користувачі, які хочуть використовувати різні теги разом, можуть накладати їх один поверх іншого, Але, порядок накладання **повинен бути в рамках дозволенного**. При неправильному використанні це може відмінити форматування.  
```

File: wiki/BBCode/uk.md [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+
+### Колір
+
+```
+[color=HEX CODE]text[/color]
+```
+
+*Щоб дізнатися весь список кольорів перейдіть за [X11 color names](https://en.wikipedia.org/wiki/X11_color_names#Color_name_chart)*
+
+Тег `[color]` використовується для змінення кольору тексту. Тег використовує [HEX code](https://en.wikipedia.org/wiki/Web_colors#Hex_triplet) формат для визначення кольору, також можна використовувати НТМL кольори такі як "red" або "green". Щоб вказати НТМL колір, замініть `HEXCODE` на ім'я НТМL кольору
```
Comment (endeshnik) [inline wiki/BBCode/uk.md:75]:
```suggestion
Тег `[color]` використовується для змінення кольору тексту. Тег використовує [HEX code](https://en.wikipedia.org/wiki/Web_colors#Hex_triplet) формат для визначення кольору, також можна використовувати НТМL кольори, такі як "red" або "green". Щоб вказати НТМL колір, замініть `HEXCODE` на ім'я НТМL кольору
```

File: wiki/BBCode/uk.md [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+
+### Цитата
+
+```
+[quote="NAME"]
+text
+[/quote]
+```
+
+Тег `[quote]` використовується для оформлення довгих цитат (a.k.a. "block quotes") за допомогою відступів, кольору, жирності, та відокремлення (за допомогою рожевої вертикальної лінії) тексту. Вміст цитата розміщений між відкриваючими і закриваючими тегами, а автор цитати записується `NAME` (проте вказувати аргумент не обов'язково).
```
Comment (endeshnik) [inline wiki/BBCode/uk.md:140]:
```suggestion
Тег `[quote]` використовується для оформлення довгих цитат (a.k.a. "block quotes") за допомогою відступів, кольору, жирності та відокремлення (за допомогою рожевої вертикальної лінії) тексту. Вміст цитати розміщений між відкриваючими і закриваючими тегами, а автор цитати записується `NAME` (проте вказувати аргумент не обов'язково).
```

File: wiki/BBCode/uk.md [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+
+### Профіль
+
+```
+[profile=userid]username[/profile]
+```
+
+Тег `[profile]` використовується щоб додати посилання на профіль користувача по його нікнейму або ID. `[profile]` і `[url]` відрізняються тим що `[profile]` відображає картку профіля коли посилання виділене.
+
+*Зауважте: ID користувача, це цифри після `/users/` в посиланні на профіль.*
```
Comment (endeshnik) [inline wiki/BBCode/uk.md:194]:
```suggestion
*Зауважте: ID користувача - це цифри після `/users/` в посиланні на профіль.*
```

File: wiki/BBCode/uk.md [hunk 6]
```diff
@@ -0,0 +1,322 @@
...
+
+Тег `[lucky]` є таким же самим як і тег `[google]`
+
+### Заголовок (v2)
+
+```
+[text]
+```
+
+Тег *Heading (v2)* це застарілий тег, який який форматував текст в химерний, рожевий, підкреслений заголовок. Тег працював тільки в форумах карт і з'являвся тільки після публікації. В тега не було власної кнопки і він був записаний за допомогою лапок
```
Comment (endeshnik) [inline wiki/BBCode/uk.md:316]:
```suggestion
Тег *Heading (v2)* є застарілим тегом, який який форматував текст в химерний, рожевий, підкреслений заголовок. Тег працював тільки в форумах карт і з'являвся тільки після публікації. В тега не було власної кнопки і він був записаний за допомогою лапок
```

**Review by endeshnik — CHANGES_REQUESTED** [review]

File: wiki/BBCode/uk.md [hunk 7]
```diff
@@ -0,0 +1,320 @@
...
+
+### Жирний шрифт
+
+```
+[b]text[/b]
+```
+
+Тег `[b]` використовується, щоб виділити текст за допомогою жирного шрифту. BBCode виділення не змінює розмір шрифту.
+
+Кнопка на панелі форматування ![Bold button](img/bold.png "Bold")
```
Comment (TPGPL) [inline wiki/BBCode/uk.md:34]: seems like you forgot to translate tooltips and alt-texts for those icons (same below)

File: wiki/BBCode/uk.md [hunk 8]
```diff
@@ -0,0 +1,320 @@
...
+
+### Жирний шрифт
+
+```
+[b]text[/b]
+```
+
+Тег `[b]` використовується, щоб виділити текст за допомогою жирного шрифту. BBCode виділення не змінює розмір шрифту.
+
+Кнопка на панелі форматування ![Жирний шрифт](img/bold.png "Bold")
```
Comment (TPGPL) [inline wiki/BBCode/uk.md:34]: tooltips are still not translated

for ref:

`alt-text`

File: wiki/BBCode/uk.md [hunk 9]
```diff
@@ -0,0 +1,320 @@
...
+
+Тег `[quote]` використовується для оформлення довгих цитат (a.k.a. "block quotes") за допомогою відступів, кольору, жирності та відокремлення (за допомогою рожевої вертикальної лінії) тексту. Вміст цитати розміщений між відкриваючими і закриваючими тегами, а автор цитати записується `NAME` (проте вказувати аргумент не обов'язково).
+
+*Зауважте: аргумент `NAME` має бути в лапках (`"`).*
+
+Довгі цитати використовуються замість однолінійних цитат в більш формальних розмовах. Але всередині форумів вони використовуються щоб відповісти іншому гравцю, що можна робити автоматично за допомогою кнопки `Quote reply` яка знаходиться в верхньому правому кутку комменту(показано нижче).
+
+Проте ця кнопка буде відображена **тільки якщо курсор наведений на коментар**.
+
+![Quote reply button](img/quotereply.png)
```
Comment (TicClick) [inline wiki/BBCode/uk.md:147]: aaand there's another tooltip without translation

**Review by TicClick — CHANGES_REQUESTED** [review]
this has to be all, highlight me on Discord once you have fixed it

**Review by TicClick — APPROVED** [review]

**Pull request merged**